02-express-tutorial/app.js

```diff
@@ -46,7 +64,24 @@ app.get('/api/v1/query', (req, res) => {
     res.json(filteredProducts);
 });
 
-app.all('*', (req, res) => {
+app.use("/api/v1/people", peopleRouter);
+
+
+app.get('/api/v1/people', (req, res) => {
+    res.json(people);
+});
+
+app.post('/api/v1/people', (req, res) => {
```

Comment:
This `app.post('/api/v1/people', ...);` block should be removed as line 67 handles this path.